src/Tests/xunit-runner/XUnitRunner.targets

```diff
@@ -50,7 +50,7 @@
       <_CurrentSDKCustomXUnitProject>%(SDKCustomXUnitProject.Identity)</_CurrentSDKCustomXUnitProject>
       <_CurrentPublishTargetFramework>%(SDKCustomXUnitProject.TargetFramework)</_CurrentPublishTargetFramework>
       <_CurrentPublishTargetFramework Condition="'$(_CurrentPublishTargetFramework)' == ''">$(SDKCustomXUnitPublishTargetFramework)</_CurrentPublishTargetFramework>
-      <_CurrentRuntimeTargetFramework>%(SDKCustomXUnitProject.RuntimeTargetFramework)</_CurrentRuntimeTargetFramework>
+      <_CurrentRuntimeTargetFramework>%(SDKCustomXUnitProject.TargetFramework)</_CurrentRuntimeTargetFramework>
```

Comment:
I'm not sure what's going on with this change.  I think I used to know what the RuntimeTargetFramework was but I've forgotten it.

Reply:
Without Will to ask, I wasn't completely sure. I needed to add the projects that target 472 as additional targets in the unittest.proj and within SDKCustomCreateXUnitWorkItemsWithTestExclusion.cs, I needed to be able to check the framework. I originally added an additional check for TF in the SDK* logic but that felt odd since there was an existing check. An alternative option here is to revert the xunitrunner.targets and add in the RuntimeTargetFramework in the unittests.proj file.  Let me try that instead.